docs/src/modules/components/ExampleCollection.js

```diff
@@ -0,0 +1,188 @@
+    tsLabel: 'View TS example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-vite',
+    src: '/static/images/examples/vite.svg',
+  },
+  {
+    name: 'styled-components',
+    label: 'View JS example',
+    tsLabel: 'View TS example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-cra-styled-components',
+    tsLink:
+      'https://github.com/mui/material-ui/tree/master/examples/material-cra-styled-components-ts',
+    src: '/static/images/examples/styled.png',
+  },
+  {
+    name: 'Gatsby',
+    label: 'View example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-gatsby',
+    src: '/static/images/examples/gatsby.svg',
+  },
+  {
+    name: 'Preact',
+    label: 'View example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-preact',
+    src: '/static/images/examples/preact.svg',
+  },
+  {
+    name: 'CDN',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-via-cdn',
+    label: 'View example',
+    src: <FilterDramaIcon />,
+  },
+  {
+    name: 'Express.js (server-rendered)',
+    label: 'View example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-express-ssr',
+    src: '/static/images/examples/express.png',
+  },
+];
+
+export default function ExampleCollection() {
+  return (
+    <Grid container spacing={2}>
+      {examples.map((example) => (
+        <Grid key={example.name} item xs={12} sm={6}>
+          <Card
+            sx={[
+              {
+                p: 2,
+                height: '100%',
+                display: 'flex',
+                alignItems: 'center',
+                gap: 2,
+                bgcolor: '#fff',
+                backgroundImage: 'none',
+                borderRadius: 1,
+                border: '1px solid',
+                borderColor: 'grey.200',
+                boxShadow: 'none',
+                '&:hover': {
+                  borderColor: 'grey.300',
+                  boxShadow: '0px 2px 8px rgba(170, 180, 190, 0.2)',
+                },
+              },
+              (theme) =>
+                theme.applyDarkStyles({
+                  bgcolor: 'transparent',
+                  borderColor: 'primaryDark.700',
+                  '&:hover': {
+                    borderColor: 'primaryDark.600',
+                    boxShadow: '0px 2px 8px rgba(0, 13, 26, 1)',
+                  },
+                }),
+            ]}
+          >
+            <Avatar
+              imgProps={{
+                width: '40',
+                height: '40',
+                loading: 'lazy',
+              }}
+              {...(typeof example.src === 'string'
+                ? { src: example.src }
+                : { children: example.src })}
+              alt={example.name}
+            />
+            <div>
+              <Typography variant="body" fontWeight={600} sx={{ fontFamily: 'IBM Plex Sans' }}>
```

Comment:
We should use a design token here. For example how it loads initially before the font fallback quicks in.

<img width="436" alt="Screenshot 2023-05-25 at 14 07 04" src="https://github.com/mui/material-ui/assets/3165635/df89b35d-8676-4fb9-a181-da6db11cce61">

Since it's using the parent theme, it's not even needed.

```suggestion
              <Typography variant="body" fontWeight={600}>
```

Then, same problem with the fontWeight, I think that it should be using the theme token:

```suggestion
              <Typography variant="body" fontWeight="semiBold">
```